lua/orgmode/colors/from_config/priority.lua

```diff
@@ -0,0 +1,12 @@
+local config = require('orgmode.config')
+
+local M = {}
+
+function M.get_queries()
+  return vim.tbl_map(function(priority)
+    return string.format('((expr) @OrgPriority%s (#eq? @OrgPriority%s "[#%s]"))', priority, priority, priority)
```

Comment:
_todo_: The [official](https://orgmode.org/manual/Priorities.html) docs denote that the priority always comes second after the "TODO state", so where we could get away with something like `(item . (expr) . (expr) ...)` to extract "the second expression" rather than comparing against _all_ expressions?